bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieServer.java

```diff
@@ -82,14 +81,6 @@ public BookieServer(ServerConfiguration conf, StatsLogger statsLogger)
             BookieException, UnavailableException, CompatibilityException, SecurityException {
         this.conf = conf;
         validateUser(conf);
-        String configAsString;
-        try {
-            configAsString = conf.asJson();
-            LOG.info(configAsString);
-        } catch (ParseJsonException pe) {
```

Comment:
Yes you are reverting a change from @charanreddy

Reply:
Ah crap, must have been in flight as I was turning the merge into a rebase. Will fix.

Reply:
updated